tools/add_to.hpp

```diff
@@ -140,52 +140,51 @@ namespace antler {
       }
 
       add_to_project(CLI::App& app) {
-         path = system::fs::current_path().string();
          subcommand = app.add_subcommand("add", "Add an app, dependency, library or test to your project.");
-         subcommand->add_option("-p, path", path, "Path containing the project's yaml file.");
+         subcommand->add_option("-p", path, "Path containing the project's yaml file.")->default_val(".");

```

Comment:
For both the add and update subcommands, I think we should remove path at this first level because the following line is valid, but it's not clear what path is chosen:
```sh
antler-proj \
   add -p ./PathOne \
   app -p ./PathTwo --name AppName
```

Something to think about: based on testing, it seems like the default value is set when the option is added, but in the _very unlikely event_ CLI11 changed this to parse time it's not clear what would happen.

Reply:
good point

Reply:
maybe we should move path to the app level?
so any path command will be like:
`antler-proj --path ./test init --name test --version 1.0.0`

Reply:
The problem with that approach becomes clear when we try this: `antler-proj init --help` and we don't get the `--path` flag.  I _think_ there is a way for subcommands to "inherit" a command from a parent. That could solve the problem.